x/leverage/keeper/collateral.go

```diff
@@ -54,10 +83,10 @@ func (k Keeper) setCollateralAmount(ctx sdk.Context, borrowerAddr sdk.AccAddress
 }
 
 // GetTotalCollateral returns an sdk.Coin representing how much of a given uToken
-// the x/leverage module account currently holds as collateral. Non-uTokens and invalid
-// assets return zero.
+// the x/leverage module account currently holds as collateral. Non-uTokens return
+// zero.
 func (k Keeper) GetTotalCollateral(ctx sdk.Context, denom string) sdk.Int {
-	if !k.IsAcceptedUToken(ctx, denom) {
+	if !types.HasUTokenPrefix(denom) {
 		// non-uTokens cannot be collateral
 		return sdk.ZeroInt()
 	}
```

Comment:
Subtle change here: If an asset were ever blacklisted while some of its uToken collateral remained in the system, that collateral would now still be visible to queries.